README.md

```diff
@@ -0,0 +1,38 @@
+# Yophpcs
+
+Yophpcs is the PHP Code Sniffer for Yola that defines the PHP Coding Standards
+
+## Using the Code Sniffer
+
+```
+$ vendor/bin/phpcs --extensions=php --standard=/path/to/yophpcs/Standard /path/to/project
+```
+
+## Yola Standard
+The base standard, it is derived from the [PSR-2 Coding Style Guide](https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md).
+
+### Differences
+
+#### 4.2 Properties
+
+Property names SHOULD be prefixed to indicate private visibility.
```

Comment:
> I like to denote private variables explicitly so that when they're used or read, it's immediately apparent that the variable is private. I do this for JS too on functions/variables that aren't intended to be called/manipulated directly.

> PSR-2 recommends against it, and it's probably the ruleset your linter is using. However, many major PHP frameworks modify PSR-2 to set the underscore on private variables. Most of my experience has been with such frameworks that did this, and I think it's a good idea, but I'm not going to be upset if it's decided against.

> In yola/yophpcs#2, the yola-specific php style guide for phpcs I worked on, it will yell at you for not having the underscore. If we don't want the underscores on private variables, we should change it there.

copy/pasting @DevShep's reply.

> I do this for JS too

Makes sense there (and python too) because variable declaration does not involve scope.

> If we don't want the underscores on private variables, we should change it

I don't feel super strong one way or another, but if I had to vote, I say no underscores.  It feels redundant. This may also be my own inexperience, but when I see PHP vars prefixed with an underscore, my first assumption is that the variable is referencing some internal that I'm unfamiliar with.